docker/driver-loader/docker-entrypoint.sh

```diff
@@ -25,4 +25,19 @@ do
     ln -s "$i" "/usr/src/$base"
 done
 
-/usr/bin/falco-driver-loader "$@"
\ No newline at end of file
+if [ -n "$HOST_ROOT" ]; then
+    echo "* Setting up /lib/modules links from host"
+    ln -s /lib/modules $HOST_ROOT/lib/modules 
```

Comment:
drivers makefile uses `/lib/modules`; in case `HOST_ROOT` is set, we link /lib/modules to its relative path under HOST_ROOT.
This was done before at docker image build time.

Reply:
I don't understand why we need this. 

Moreover, `$HOST_ROOT` can be equal to `/` which would produce:
```
ln -s /lib/modules //lib/modules
```


Here's a legit use case for `/`:
```
docker pull falcosecurity/falco-no-driver:latest
docker run --rm -i -t \
    -e HOST_ROOT=/ \
    --cap-add SYS_PTRACE --pid=host $(ls /dev/falco* | xargs -I {} echo --device {}) \
    -v /var/run/docker.sock:/var/run/docker.sock \
    falcosecurity/falco-no-driver:latest
```


Ref https://falco.org/docs/getting-started/running/#docker-least-privileged

Reply:
Yep, the issue here is when you have an HOST_ROOT that is not "/host". Before, you'd have "/host/lib/modules" linked to "/lib/modules" from Dockerfile, even when, at runtime, you'd got HOST_ROOT="/foo".
So, you'd have a Falco running with HOST_ROOT="/foo", but /lib/modules/ symlinked to "/host/lib/modules".
Of course, this ain't gonna work :)

Reply:
You are right, i think we should update the 
`if [ -n "$HOST_ROOT" ]; then`
with
`if [[ -n "$HOST_ROOT" -a "$HOST_ROOT" != "/" ]]; then`

Reply:
Fixed!